src/results/ResultsContainer.jsx

```diff
@@ -1,31 +1,55 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
 import { fetchResults } from './state/actions';
 import ResultsTable from './ResultsTable';
+import { serializableDeepCopy } from '../utils/utils';
+import { isClauseTouched } from '../advanced-search/utils/clause';
 
 export class Results extends Component {
   componentDidMount() {
-    const { query, dispatchFetchResults } = this.props;
-    dispatchFetchResults(query);
+    console.log('here');
+    const { queryClauses: queryClausesFromProps, dispatchFetchResults, location } = this.props;
+    const touchedQueryClausesFromProps = queryClausesFromProps.filter(isClauseTouched);
```

Comment:
Likewise, maybe 'changedQueryClausesFromProps'?

Reply:
Logic refactored so that detecting if the user has touched a clause is no longer needed.